### ISSUE TITLE
Stream.download throw an excepation when elapsed variable is zero

### ISSUE THREAD

**Issue by dck** [np1/pafy#80]
If I download stream in less than 1 second I get `ZeroDivisionError` on this [line](https://github.com/np1/pafy/blob/master/pafy/pafy.py#L962):
```
926: rate = ((bytesdone - offset) / 1024) / elapsed
```

**Comment by efulmo** [comment]
Yep. Confirm. It happens for me sometimes

**Comment by ids1024** [comment]
Can you name a video with this issue?

**Comment by efulmo** [comment]
```python
>>> v=pafy.new("https://www.youtube.com/watch?v=olstL16w2oY")
>>> s=v.audiostreams
>>> s[0].download()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "d:\work\python\virtualenv\testenv\lib\site-packages\pafy\pafy.py", line
962, in download
    rate = ((bytesdone - offset) / 1024) / elapsed
ZeroDivisionError: float division by zero
```

**Comment by ids1024** [comment]
This should be fixed by commit 9c612d277b422dbfbd214a8c6435398aaf236f00 in the dev version. The bug should be gone in the next release.
